etc/START.htm

```diff
@@ -83,11 +83,11 @@
         <li><a href="http://localhost:2316/de" target="_blank">Verwaltung und Erstellung</a></li>   
       </ul>
       <ul class="doc">
-        <li><a href="http://opensource.geneanet.org/projects/geneweb/wiki/en_home" target="_blank">Bedienungsanleitung</a></li>
+        <li><a href="http://geneweb.org" target="_blank">Bedienungsanleitung</a></li>
       </ul>
       <div class="bottom">
-        <a href="http://opensource.geneanet.org/projects/geneweb/wiki/Licence" target="_blank">Lizenz</a> /
-        <a href="http://opensource.geneanet.org/projects/geneweb/wiki/Credits" target="_blank">Impressum</a>
+        <a href="http://geneweb.org" target="_blank">Lizenz</a> /
```

Comment:
The license used of the wiki (CC BY-SA 3.0 on footer) is not the one of GeneWeb itself.